config/intellij-idea-inspections.xml

```diff
@@ -3085,7 +3086,8 @@
                    enabled_by_default="true"/>
   <inspection_tool class="MethodOnlyUsedFromInnerClass" enabled="true" level="ERROR"
                    enabled_by_default="true">
-    <option name="ignoreMethodsAccessedFromAnonymousClass" value="false"/>
+    <!-- Better to keep anonymous classes as small as possible. -->
+    <option name="ignoreMethodsAccessedFromAnonymousClass" value="true"/>
```

Comment:
> TreeTable.java (1) expandSelectedNode() Method expandSelectedNode()

our code:
https://github.com/checkstyle/checkstyle/blob/591b41028cecc786ea94556791a8e53a99defb4c/src/main/java/com/puppycrawl/tools/checkstyle/gui/TreeTable.java#L115-L133

I am ok with this change.